src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "int",
+             "default": -1,
+             "description": "Channel number. If -1, use the first channel",
+             "name": "option"
```

Comment:
@edersondisouza @laykuanloon I changed the sol_iio_open/close API, Using a list to trace these opened devices, Please help to review. The test sensor I am using does support two different channels, but it does not support buffer reading. I will find another one...